datalad/core/distributed/clone.py

```diff
@@ -926,8 +926,16 @@ def decode_source_spec(spec, cfg=None):
         dsid, version = source_ri.fragment.split('@', maxsplit=1) \
             if '@' in source_ri.fragment else (source_ri.fragment, None)
         uuid_regex = r'[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}'
-        if not re.match(uuid_regex, dsid):
-            raise ValueError('RIA URI does not contain a valid dataset ID: {}'.format(spec))
+        if re.match(uuid_regex, dsid):
+            trace = '{}/{}'.format(dsid[:3], dsid[3:])
+            default_destpath = dsid
+        elif dsid.startswith('~'):
+            trace = 'alias/{}'.format(dsid[1:])
```

Comment:
since this is more of an internal purpose, why not to have it as `.alias/` or even `.ria/aliases/` or alike?  then you could still have an `alias` dataset (happen ria allow it at some point)

Reply:
I think it is fine to not have further obfuscation. Having a dataset stored in a directory named like this would represent a fundamental concept change. I doubt that we would keep the name or URL scheme at that point.